loophp/psr17/1.0/config/packages/loophp_psr17.yaml

```diff
@@ -0,0 +1,9 @@
+services:
+    # Register loophp/psr17/Psr17 class and autowire/autoconfigure it.
+    loophp.psr17.psr17:
```

Comment:
Is there a reason you don’t use the class name as id?

Reply:
True, I will update!

Reply:
Updated!